streetscape-version-1-overview.md

```diff
@@ -0,0 +1,82 @@
+# Streetscape 1
```

Comment:
is it worth aligning with the version specified in the Streetscape repo [here](https://github.com/NYCPlanning/ae-streetscape/blob/main/package.json#L3) or does it seem worth having a different version for the design half of Streetscape?

noting that the streetscape repo gets packaged and pushed to npm [here](https://www.npmjs.com/package/@nycplanning/streetscape) so that it's components can be use in apps [like this](https://github.com/NYCPlanning/ae-cp-map/blob/main/app/components/CapitalProjectDetailPanel/CapitalProjectDetailPanel.tsx#L5-L15):

```ts
import {
  Box,
  Flex,
  Heading,
  HStack,
  Text,
  Wrap,
  WrapItem,
  IconButton,
  Hide,
} from "@nycplanning/streetscape";
```

Reply:
it depends since streetscape on the design front involves parts that are not only targeted to implementation. it would keep everything consistent, it also depends how ae-streetscape will be maintained in the future (will designers handle more ae-streetscape maintance in the future or is that something that has to be translated through ae first). 

also brings into question how often will streetscape will be updated.